src/app/clusters/mode-select-server/mode-select-server.cpp

```diff
@@ -28,12 +28,22 @@
 #include <app/clusters/mode-select-server/supported-modes-manager.h>
 #include <app/util/af.h>
 #include <app/util/attribute-storage.h>
+#include <app/util/error-mapping.h>
+#include <app/util/util.h>
 #include <lib/support/CodeUtils.h>
 
 using namespace std;
 using namespace chip;
 using namespace chip::app;
 using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::ModeSelect;
+using namespace chip::Protocols::InteractionModel;
+
+#ifndef IGNORE_MODE_SELECT_CLUSTER_START_UP_MODE
```

Comment:
Who would define this and where?  Should this be a CHIP_CONFIG setting?

Reply:
- I followed the current pattern in [level-control.cpp](https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/level-control/level-control.cpp#L74) and [on-off-server.cpp](https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/on-off-server/on-off-server.cpp#L213). I assumed there is a mechanism to manage these flags. Let me know this is not the way.

Reply:
There might have been in the original Silicon Labs SDK, but I don't think there is in ours....  @selissia @jmartinez-silabs thoughts?

Reply:
This is a leftover from Silabs cluster code where, yes, there was a mechanism to generate these flags based on user selections in a cluster configuration GUI. This is not applicable in Matter and we should not be using these flags here or in level-control.cpp and on-off-server.cpp